api/config/config_staging.json

```diff
@@ -116,8 +116,7 @@
     "enablePrivateDataprocWorker": true,
     "ccSupportWhenAdminLocking": false,
     "enableMultiReview": false,
-    "enableUpdatedDemographicSurvey": true,
-    "enableDrugWildcardSearch": true
```

Comment:
Story only says to remove references to flag, but not the flag itself. Please revert.